site/en/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,1356 @@
+        "  tf.keras.layers.GlobalAveragePooling2D(),\n",
+        "  tf.keras.layers.Dense(len(label_names))])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "foQYUJs97V4V",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Now it produces outputs of the expected shape:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "1nwYxvpj7ZEf",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "logit_batch = model(image_batch).numpy()\n",
+        "\n",
+        "print(\"min logit:\", logit_batch.min())\n",
+        "print(\"max logit:\", logit_batch.max())\n",
+        "print()\n",
+        "\n",
+        "print(\"Shape:\", logit_batch.shape)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "pFc4I_J2nNOJ",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Compile the model to describe the training procedure:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "ZWGqLEWYRNvv",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "model.compile(optimizer=tf.train.AdamOptimizer(), \n",
+        "              loss=tf.keras.losses.sparse_categorical_crossentropy,\n",
+        "              metrics=[\"accuracy\"])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "tF1mO6haBOSd",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "There are 2 trainable variables: the Dense `weights` and `bias`:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "pPQ5yqyKBJMm",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "len(model.trainable_variables) "
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "kug5Wg66UJjl",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "model.summary()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "f_glpYZ-nYC_",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Train the model.\n",
+        "\n",
+        "Normally you would specify the real number of steps per epoch, but for demonstration purposes only run 3 steps."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "AnXPRNWoTypI",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "model.fit(ds, epochs=1, steps_per_epoch=3)  # steps_per_epoch=math.ceil(len(all_image_paths)/BATCH_SIZE)"
```

Comment:
It doesn't exist yet, but I think the 2.0 recommendation for this will be `model.fit(ds.take(3), epochs=1)` (when the dataset input is finite).

Reply:
Great. LMK when that happens and I can scrub `steps_per_epoch` from our docs.